### PR TITLE
mimic: qa: reduce slow warnings arising due to limited testing hardware

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/26903
http://tracker.ceph.com/issues/24841

### CODE REVIEW

**Comment by smithfarm** [comment]
@batrick Cherry-picked follow-on fix from https://github.com/ceph/ceph/pull/23542

**Comment by batrick** [comment]
conflict resolution LGTM

**Comment by yuriw** [comment]
[wip-yuri5-testing-2018-09-05-1625-mimic](https://trello.com/c/xwoIGoxK/390-wip-yuri5-testing-2018-09-05-1625-mimic)

File: qa/suites/fs/bugs/conf [hunk 1]
```diff
@@ -0,0 +1 @@
+.qa/cephfs/conf
```
Comment (batrick) [inline qa/suites/fs/bugs/conf]: This sub-suite does not exist in Mimic. The QA test for https://github.com/ceph/ceph/pull/22139 was chosen not to be backported.

**Comment by yuriw** [comment]
@smithfarm pls add tags when fixed

File: qa/suites/fs/bugs/conf (hunk 1 above)
Comment (smithfarm) [reply qa/suites/fs/bugs/conf]: @batrick Thanks; dropped.

**Pull request merged**